File: app/src/main/java/ru/mamykin/foreignbooksreader/presenters/SplashPresenter.java
package ru.mamykin.foreignbooksreader.presenters;

import ru.mamykin.foreignbooksreader.views.SplashView;

public class SplashPresenter extends BasePresenter<SplashView> {
    @Override
    public void attachView(SplashView view) {
        super.attachView(view);

        view.startApplication();
    }
}

File: app/src/main/java/ru/mamykin/foreignbooksreader/presenters/SplashPresenter.kt
package ru.mamykin.foreignbooksreader.presenters

import ru.mamykin.foreignbooksreader.views.SplashView

class SplashPresenter : BasePresenter<SplashView>() {
    override fun attachView(view: SplashView) {
        super.attachView(view)

        view.startApplication()
    }
}